### PR TITLE
Create a new bio for Jennifer

### PR DESCRIPTION
**Pull request by JenniferLu**
Test

### CODE REVIEW

File: doc/jennifer-bio.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Working on my bio
```
Comment (JenniferLu) [inline doc/jennifer-bio.md:1]: Add needed bio

Comment (patrickjmarion) [inline doc/jennifer-bio.md:1]: Terrific Jennifer!  I love it :smile:

Comment (dlee01) [inline doc/jennifer-bio.md:1]: Looks like you're missing Name, Occupation, and Location.

**Comment by JenniferLu** [comment]
Looks good to me.
:+1: 
:shipit:

**Pull request merged**